pkg/controllers/state/statenode.go

```diff
@@ -221,6 +222,19 @@ func (in *StateNode) Initialized() bool {
 	return true
 }
 
+// Disruptable returns if a node has been initialized long enough for Karpenter to disrupt it
+func (in *StateNode) Disruptable(clock clock.Clock) bool {
```

Comment:
I like the naming, but because we are going to use this with emptiness, I wonder if we have to rename this to something else like, "WithinInitializionWindow()" or something more verbose

Reply:
i can see the case for this if we have more Disruptable requirements. Not sure I understand why reasoning about emptiness motivates you to think of another name.